src/script/api/script_client.hpp

```diff
@@ -24,7 +24,7 @@ class ScriptClient : public ScriptObject {
 public:
 
 	/** Different constants related to ClientID. */
-	enum ClientID : uint32_t {
+	enum ClientID : int32_t {
```

Comment:
Why? The one in network_type.h and company_cmd.h haven't been touched changed.

Reply:
Probably an over-reach :)